sessions_core/01_introduction.qmd

```diff
@@ -0,0 +1,751 @@
+---
+title: Introduction to R
+description: Your first steps in R. Learn your way around Rstudio, and meet some common R objects.
+date: last-modified
+# image: 
+params: 
+  solutions: FALSE
+  sub_title: "**------ With Solutions ------**"
+categories:
+  - Core
+  - Rstudio interface
```

Comment:
perhaps too specific

Reply:
Agree - perhaps can just simplify to "RStudio" ?